### ISSUE TITLE
Docs for versions less than 0.14

### ISSUE THREAD

**Issue by rmcd-mscb** [hyriver/pynhd#65]
### Is your feature request related to a problem?

Hi Taher, I have several projects that are limited to shapely < 2.0.  Would it be possible to also provide documentation prior to the latest release of hyriver packages?  Thanks for considering - Rich

**Comment by cheginit** [comment]
Hi Rich. Yes, there's documentation for prior version, they're on the `readthedoc` version of the website that you can find [here](https://hyriver.readthedocs.io/en/v0.13.1/). Considering that there's one website for nine packages, it's difficult to handle a versioned documentation for each new release of each package. So, every minor version update I add a new tag to the monorepo, to at least have documentation for each minor release.

BTW, let me know if you need help with migrating to `shapely` 2.0. Since `pygeos` will not release new versions anymore, it's best if you start thinking about the migration.

**Comment by cheginit** [comment]
Right, I've been meaning to migrate the versioned documents to the new website but haven't had the time. Hopefully soon!

Regarding GeoConnex, they made some changes to the service, so I rewrote the whole function in v0.14, and I am not sure if v0.13 still works. You can give it a try, though. You can check out the [example](https://docs.hyriver.io/examples/notebooks/geoconnex.html) notebook for GeoConnex.
